server/config/db.js

```diff
@@ -0,0 +1,12 @@
+/*
+Author: Multiverse Design
+Date: Nov-11-2021
+FileName : db.js
+Where to connect from
+*/
+
+
+module.exports = {
+    "URI": "mongodb://localhost/survey"
```

Comment:
"URI": mongodb+srv://multiadmin:MakeItSo99@mongodbserver.0vv2y.mongodb.net/surveys?retryWrites=true&w=majority";"

Reply:
I have updated mongo Atlas and pushed to my branch.